src/http/api/webhook.controller.ts

```diff
@@ -93,4 +93,42 @@ export class WebhookController {
             status: 200,
         });
     }
+
+    /**
+     * Handle a post.published.edited webhook
+     *
+     * @param ctx App context instance
+     */
+    async handlePostUpdated(ctx: AppContext) {
+        let data: PostInput;
+
+        try {
+            data = PostPublishedWebhookSchema.parse(
+                (await ctx.req.json()) as unknown,
+            ).post.current;
+        } catch (err) {
+            if (err instanceof Error) {
+                return BadRequest(`Could not parse payload: ${err.message}`);
+            }
+            return BadRequest('Could not parse payload');
+        }
+
+        const account = ctx.get('account');
+
+        const result = await this.postService.updateGhostPost(account, data);
+
+        if (isError(result)) {
+            const error = getError(result);
+            switch (error) {
+                case 'post-not-found':
+                    return NotFound('Could not find post to update');
```

Comment:
Do we want to create the post in this case?

Reply:
Good point. Added here: https://github.com/TryGhost/ActivityPub/pull/1011/commits/7301d1306f9494d0012d15652799228ab1804be8